stl/debugger/STL.natvis

```diff
@@ -1827,4 +1827,14 @@ SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
     </Expand>
   </Type>
 
+  <Type Name="std::source_location">
+    <DisplayString>line: {_Line}, column: {_Column}</DisplayString>

```

Comment:
Perhaps?
```suggestion
    <DisplayString>{_Function,sb}: ({_Line}, {_Column})</DisplayString>
```

Reply:
I agree that I prefer the second option more